core/src/main/scala/pureconfig/package.scala

```diff
@@ -278,8 +278,8 @@ package object pureconfig {
       files
         .map(parseFile)
         .foldLeft[Either[ConfigReaderFailures, Seq[TypesafeConfig]]](Right(Seq())) {
-          case (c1, c2) =>
-            ConfigConvert.combineResults(c1, c2)(_ :+ _)
+          case (c1, Left(ConfigReaderFailures(_: CannotReadFile, Nil))) => c1
```

Comment:
Do you think we should look for `CannotRedFile` anywhere in the list of failures rather than only exactly a list of that one failure? It seems a bit more robust to me, if highly paranoid.